Email Template/email.html

```diff
@@ -0,0 +1,512 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>
+        EMAIL CONTENT
+    </title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lora&family=Montserrat:wght@100;400;700&family=Oswald:wght@200;400&family=Raleway:wght@700&display=swap"
+        rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lora&family=Montserrat:wght@100;400;700&family=Nunito+Sans&family=Oswald:wght@200;400&family=Raleway:wght@700&display=swap"
```

Comment:
Check this article on how fonts in email works -> https://campaignmonitor.com/resources/guides/web-fonts-in-email/

Check what the template email does for fonts, and see how it appears when you copy paste inside gmail.